engines/stark/ui/menu/mainmenu.cpp

```diff
@@ -181,8 +172,24 @@ void MainMenuScreen::settingsHandler() {
 	StarkUserInterface->changeScreen(Screen::kScreenSettingsMenu);
 }
 
+void MainMenuScreen::boxHandler() {
+	if (!StarkSettings->isDemo() && StarkSettings->hasBookOfSecrets()) {
+		StarkUserInterface->changeScreen(kScreenGame);
+		StarkResourceProvider->initGlobal();
+		StarkScene->setFadeLevel(1.0f);
```

Comment:
The "Book of Secrets" will not handle the fade level by itself. I don't find a problem setting it here before the actual game starts.

Reply:
Shouldn't the fade level be initialized to 1.0 in the Scene class rather than here?